packages/release/prepare-local-fs.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Prepare Local Filesystem (/local)
+DefaultDependencies=no
+Wants=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
+After=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
+Before=local.mount
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+
+# Create the filesystem on the partition, if it doesn't exist.
+ExecStart=/usr/lib/systemd/systemd-makefs ext4 /dev/disk/by-partlabel/BOTTLEROCKET-DATA
+# Give the filesystem volume a shorter label name
+ExecStart=/sbin/tune2fs -L BR-DATA /dev/disk/by-partlabel/BOTTLEROCKET-DATA
```

Comment:
It's probably not worth doing this since we'd end up doing it on every boot, and `tune2fs` actually relabels the filesystem which causes udev link churn (granted we don't look at `/dev/disk/by-label/BR-DATA` anywhere today).